firebase-database-collection/firebase-database-collection.gradle

```diff
@@ -22,8 +22,11 @@ android {
 }
 
 dependencies {
+    implementation "com.google.android.gms:play-services-base:$playServicesVersion"
```

Comment:
This adds an indirect dependency on the Android Support Annotations package that includes `@RestrictTo`. Rather than including it directly (and keep updating the version as Play Services updates its dependencies), I pulled in the play-services-base package which is already used by both Firestore and RTDB.